test/functional/inspec_exec_test.rb

```diff
@@ -178,6 +178,25 @@ def stderr
     assert_exit_code 100, out
   end
 
+  it "executes only specified controls when selecting the controls by literal names" do
+    inspec("exec " + File.join(profile_path, "filter_table") + " --no-create-lockfile --controls foo")
```

Comment:
Why did you choose the filter_table fixture? I don't think this problem has anything to do with FilterTable. I would suggest either finding a different profile, or creating a new one just for this purpose.

Reply:
Yes after checking the impact decided to remove this from here. I thought since we are using --controls option we are using with this we can use it. But yes definitely this is not the right place to have this. Making the required changes.